src/esm_runscripts/helpers.py

```diff
@@ -136,6 +137,78 @@ def assemble_log_message(
     return line
 
 
+def update_reusable_files(config, reusable_filetypes=None):
+    """
+    Reusable files, in general (``config["general"]["reusable_filetypes"]``), or model
+    specific (``config[<model>]["reusable_filetypes"]``) forces files of a given type,
+    such as ``bin``, ``input``, ``forcing``, etc., to be copied only once into the
+    general experiment folder, and then **reused** them through out every run. If a
```

Comment:
Isn't this supposed to be the default behavior for some of the filetypes?

Reply:
Yes, it is the default for `src` and `bin`, but you can redefine `reusable_filetypes` from the runscript, either in `general` or in the model sections.